extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -44,6 +44,9 @@ export class BookTocManager implements IBookTocManager {
 		this._sourceBook = sourceBook;
 		this.newSection = {};
 		this.tableofContents = [];
+		this._movedFiles = new Map<string, string>();
```

Comment:
Why did you move this? If you just initialize like you had before that's doing the exact same thing

Reply:
I'm assuming this is what your statement is about : 

`Initialize the movedFiles, modifiedDirectories and tocFiles maps every time we try to edit a book.`

But this is only going to do that when we construct a new BookTocManager...at which point what you had before is also going to be doing that.